app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -144,7 +144,9 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler,
     protected val browserInteractor: BrowserToolbarViewInteractor
         get() = _browserInteractor!!
 
-    private var _browserToolbarView: BrowserToolbarView? = null
+    @VisibleForTesting
+    @Suppress("VariableNaming")
+    internal var _browserToolbarView: BrowserToolbarView? = null
```

Comment:
I didn't see the warning before maybe when the variable was introduce we didn't have the check and now it's showing  because of the change to `internal`

Reply:
Yes, exactly, was suprised too :).